gemrb/GUIScripts/GUISTORE.py

```diff
@@ -1738,6 +1738,8 @@ def SetupItems (pc, Slot, Button, Label, i, storetype, idx, steal=0):
 				#containers are always clickable
 				Button.SetState (IE_GUI_BUTTON_ENABLED)
 			else:
+				color = {'r': 61, 'g': 47, 'b': 24, 'a': 100}
+				Button.SetBorder (2, color, 1,1)
```

Comment:
One other thing. Does it still work properly (no overlay) when you scroll the item list or change to a character that can use the item?

Reply:
Is only applied on the right side, on the item that you own. Is the ones that they will not buy, when the character can't use them is already implemented on the red. Scrolling work good. Don't know how to test the other part, also I mess up my local repo and after update latest master can't talk to anybody that segfault. #1484

Reply:
ok, then it must reset correctly (elsewhere), since you say scrolling works fine. I was worried the overlay would persist on other items that are sellable, since you only explicitly enable it.